README.md

```diff
@@ -37,17 +36,78 @@ Reverse geocoding is the opposite: returning a list of places near a given point
+* [query](https://github.com/pelias/query): This is where most of our actual Elasticsearch query generation happens.
+* [config](https://github.com/pelias/config): Pelias is very configurable, and all of it is driven from a single JSON file which we call `pelias.json`. Tools for reading, validating, and working with this configuration live here and are used by almost every other Pelias component
+* [dbclient](https://github.com/pelias/dbclient): A Node.js stream library for quickly and efficiently indexing records into Elasticsearch
+
+#### Helpful tools
+
+Finally, while not part of Pelias proper, we have built several useful tools for working with and testing Pelias
+
+Notable examples include:
+
+* [acceptance-tests](https://github.com/pelias/acceptance-tests): A Node.js command line tool for testing a full planet build of Pelias and ensuring everything works. Familiarity with this tool is very important for ensuring Pelias is working. It supports all Pelias features and has special facilities for testing autocomplete queries.
+* [compare](https://github.com/pelias/compare): A web-based tool for comparing different instances of Pelias (for example a production and staging environment). We have a reference instance at [pelias.github.io/compare/](http://pelias.github.io/compare)
+* [dashboard](https://github.com/pelias/dashboard): Another web-based tool for providing statistics about the contents of a Pelias Elasticsearch index such as import speed, number of total records, and a breakdown of records of various types.
+
 
-Sure! Our API lives at [search.mapzen.com](http://search.mapzen.com/), and is usable with an API key ([register here](https://mapzen.com/developers)) and generous
-rate-limits. The endpoints are documented [here](https://mapzen.com/documentation/search); happy
-geocoding!
+<hr>
 
-[The Mapzen Search documentation](https://mapzen.com/documentation/search) also applies to standalone versions of Pelias, leaving aside API keys, privacy flags, and data sources which may be configured differently for other installations.
+Here's an old diagram of how the Pelias components interact:
+
+![A diagram of the Pelias architecture.](https://cloud.githubusercontent.com/assets/4467604/6944539/3b1cdd0e-d862-11e4-995d-0b376caacad6.png)
+
+Note it doesn't include most of the frontend services. We would love help making a new diagram.
+
+
+### Documentation
+
+The main documentation lives in the [pelias/documentation](https://github.com/pelias/documentation) repository.
+
+Additionally, the README file in each of the component repositories listed above provides more detail on that piece.
+
+<details>
+  <summary>Here's an example API response from Pelias so you can see what sort of data is returned</summary>
 
 ```javascript
```

Comment:
Add comment that this is a reverse geocoding for the lat/lon of a restaurant in New York